paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -227,6 +228,8 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
                 }
             }
         }
+
+        buttonContainer.isVisible = true
```

Comment:
why is this needed? from the recording seems the button container is never hidden